e2e.groovy

```diff
@@ -0,0 +1,57 @@
+def cico_retries = 16
+def cico_retry_interval = 60
+def ci_git_repo = 'https://github.com/Yuggupta27/ceph-csi'
```

Comment:
I am not sure when we will do this repo swtiching though :)

Reply:
Since it is a work in progress, it is set to the personal repo for testing. As soon as we reach a final stable pipeline, we will update it to ceph-csi repo.

Reply:
Once you address the remaining comments, you can change this to the normal repository too. After that we can get this merged, and run some more manual CI jobs before adding the .yaml configuration under `jobs/`.